adventure.js

```diff
@@ -0,0 +1,28 @@
+var kind = prompt('What kind of egg are you? Animal, Edible, or Other');
+
+if (kind === 'Animal') {
+  animal = prompt('Are you Magical or Scientific?');
+    if (animal === 'Magical') {
+      alert("You're a Golden Goose Egg!");
+    } else {
+      alert("You're a Dinosaur Egg!");
+    }
+}
+else if (kind === 'Edible') {
```

Comment:
I'd bring this up the the line above:

`} else if (...) {`

That way you know which `if` this `else` is paired with